src/Etsy/methods.json

```diff
@@ -759,9 +759,9 @@
     "params": {
       "listing_id": "int",
       "products": "stringJSON",
-      "price_on_property": "array(int)",
-      "quantity_on_property": "array(int)",
-      "sku_on_property": "array(int)"
+      "price_on_property": "string",
```

Comment:
No, there is another issue here. I believe that etsy is documenting this unclearly. The accepted format is a string with comma separated integers (which works 100%). Passing array with results in OAuth PHP segmentation faults with all the latest updates and everything. Not sure is someone else is seeing this. When I get a chance I'll write a test for this.